docs/data/sdks/typescript-react-native/index.md

```diff
@@ -743,3 +754,13 @@ The React Native SDK does not directly access the IDFA as it would require addin
 Here is an [example Plugin that sets the IDFA](https://github.com/amplitude/Amplitude-TypeScript/blob/main/examples/plugins/react-native-idfa-plugin/idfaPlugin.ts)  using a third-party library.
 
 --8<-- "includes/abbreviations.md"
+
+### Over the air updates (OTA)
+
+If you are using platform like Expo that supports OTA updates. It is important to know our SDK has both native and JS code. If you are using OTA updates, you will need to make sure the native code is updated as well. See Expo's documentation on [publishing](https://docs.expo.dev/archive/classic-updates/publishing) and [runtime versions](https://docs.expo.dev/eas-update/runtime-versions/) for more details.
```

Comment:
I didn't get the meaning of this actually. Does that mean, we need to use 1.3.0 while using EXPO?

Reply:
The versions in the list have changes to the native code. Other versions will handle OTA updates without issues.
So if a customer is using `1.1.0` and then want to update to `1.3.0` they can't do it via OTA. But updating to `1.1.0` -> `1.2.0` would be fine. OR from `1.3.0` -> `1.4.0` etc

Reply:
Thanks for the explanation.